library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,61 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+For any particular library dependency chain, the libraries involved need to
+coordinate the adoption of new major versions of any shared dependency that has
+breaking changes. This is necessary so that consumers can easily use the latest
+version of all of the functionality together.  This best practice doesn't apply
+to the adoption of new major versions where the package was renamed, because the
+old and new packages can co-exist.
+
+Below is an example of what happens when a new major version of a leaf library
+is partially adopted. Start with the following dependency chain (both gax-java
```

Comment:
"Start with the following dependency chain (both " --> Suppose

Reply:
Will do